libs/types-common/src/Util/Options.hs

```diff
@@ -0,0 +1,43 @@
+module Util.Options where
+
+import Data.Aeson (FromJSON)
+import Data.Maybe (fromMaybe)
+import Data.Monoid
+import Data.Yaml (decodeFileEither, ParseException)
+import Options.Applicative
+import System.Directory
+import System.Environment (getArgs)
+import System.IO (hPutStrLn, stderr)
```

Comment:
Can we instead use our logger as, for instance, [here](https://github.com/wireapp/wire-server/blob/develop/services/brig/src/Brig/API/User.hs#L115) which logs messages in a format that we can already parse?

Reply:
This is intended to be used before or without configuring the logging framework. This is basically a temporary workaround, until we decide for either options or config files. When we have one way to configure the app, we will simply terminate it with an error as before this PR.

I would rather remove "logging" from this file, as it is not really an error anyway.

Reply:
OK, let's leave it as-is then 👍